internal/artifacts/module_arch_test.go

```diff
@@ -466,6 +478,19 @@ module-types:
 			Ω(err).Should(Succeed())
 		})
 	})
+
+	var _ = DescribeTable("isArchive", func(path string, expectedResult, expectedError bool) {
+		res, err := isArchive(path)
+		if expectedError {
+			Ω(err).Should(HaveOccurred())
+		} else {
```

Comment:
I think you should check both in any case - if expectedError is false check it didn't occur, and check the expected result anyway (since it's used also when there's an error)